plugins/dominant-color-images/tests/test-dominant-color-image-editor-imagick.php

```diff
@@ -33,4 +35,164 @@ static function ( $editor ): bool {
 			}
 		);
 	}
+
+	/**
+	 * @covers Dominant_Color_Image_Editor_Imagick::get_dominant_color
```

Comment:
Instead of repeating `Dominant_Color_Image_Editor_Imagick` with each test, you can add to the class itself:

```
@coversDefaultClass Dominant_Color_Image_Editor_Imagick
```

And then this can be simply:

```suggestion
	 * @covers ::get_dominant_color
```